TechnicoBackend/Controllers/UserController.cs

```diff
@@ -0,0 +1,41 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using TechnicoBackend.Services;
+using TechnicoBackend.Models;
+
+namespace TechnicoBackend.Controllers
+{
+    [ApiController]
+    [Route("api/[controller]")]
+    public class UserController : ControllerBase
+    {
+        private readonly UserService _userService;
```

Comment:
Consider initializing an IUserService here instead of UserService, which would be mapped accordingly using proper Dependency Injection in Program.cs.